src/dstack/_internal/server/routers/pools.py

```diff
@@ -110,7 +110,7 @@ async def show_pool(
 async def add_instance(
     body: AddRemoteInstanceRequest,
     session: AsyncSession = Depends(get_session),
-    user_project: Tuple[UserModel, ProjectModel] = Depends(ProjectMember()),
+    user_project: Tuple[UserModel, ProjectModel] = Depends(ProjectAdmin()),
```

Comment:
Is it an admin-level operation? Users can submit runs, but cannot create an instance beforehand. Isn't it strange?